src/schema/v2/me/auction_results_by_followed_artists.ts

```diff
@@ -103,14 +103,21 @@ const AuctionResultsByFollowedArtists: GraphQLFieldConfig<
         ({ total_count, _embedded }) => {
           const totalPages = Math.ceil(total_count / size)
 
+          // filter items without artist id
+          const filteredAuctionResults = _embedded.items.filter(
+            (auctionResult) => auctionResult.artist_id
+          )
```

Comment:
We can't guarantee that auction results coming from `Diffusion` have an `artist_id`. E.g. the following lot (auctionResult) let's the query fail:

```
"title": "Kubistisch Stilleven",
"id": 338102
```